src/config.py

```diff
@@ -45,25 +45,34 @@ class TestingConfig(DefaultConfig):
 
 class StagingConfig(DefaultConfig):
     TESTING = False
+    DEBUG = False
+    SQLALCHEMY_DATABASE_URI = f"postgresql+psycopg2://" \
+                              f"{os.getenv('DB_USERNAME')}:" \
+                              f"{os.getenv('DB_PASSWD')}@" \
+                              f"{os.getenv('DB_ENDPOINT')}:" \
+                              f"5432/" \
+                              f"{os.getenv('DB_NAME')}"
 
 
 class ProductionConfig(DefaultConfig):
     TESTING = False
     DEBUG = False
-
-    SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2:" \
-                              "//teste:teste@127.0.0.1:54320/hummingbird-v2"
+    SQLALCHEMY_DATABASE_URI = f"postgresql+psycopg2://" \
+                              f"{os.getenv('DB_USERNAME')}:" \
+                              f"{os.getenv('DB_PASSWD')}@" \
+                              f"{os.getenv('DB_ENDPOINT')}:" \
+                              f"5432/" \
+                              f"{os.getenv('DB_NAME')}"
     # SWAGGER
     SWAGGER_SUPPORTED_SUBMIT_METHODS = []
 
     # AUTHORIZATION
-    # TODO: Create these credentials
-    # AUTH_USERNAME = ''
-    # AUTH_PASSWORD = ''
+    AUTH_USERNAME = 'hummingbird_p'
+    AUTH_PASSWORD = '88FDD586EF3A7'
```

Comment:
não conseguimos colocar isso como secrets?

Reply:
Você diz como variável de ambiente? Se for, é so me passar o nome que eu ja adiciono e atualizo o PR.

Reply:
pode colcoar esses mesmos nomes, vou adicionar no task_definition

Reply:
Blz

Reply:
@thiagolsfortunato, feito